### ISSUE TITLE
Buzz will not open after download "developer can not be confirmed" 

### ISSUE THREAD

**Issue by TanteGerd** [chidiwilliams/buzz#271]
I read through the first post on "issues", but it seems like this is a slightly different problem: I have downloaded Buzz on my MacBook Pro, but when I try to open it Im told that "Buzz can not be opened because the developer can not be confirmed. macOS can not confirm that this program does not contain malware." How do I work my way around this? Thanks!

**Comment by chidiwilliams** [comment]
What version of Buzz do you have downloaded?

**Comment by TanteGerd** [comment]
Version 0.6.4

**Comment by chidiwilliams** [comment]
Yes, I've just noticed that I've broken the code-signing on the Mac build. Will take a look at this.

**Comment by TanteGerd** [comment]
Thank you!!

**Comment by chidiwilliams** [comment]
Does this work for you? https://github.com/chidiwilliams/buzz/releases/download/v0.7.0/Buzz-0.7.0-mac.dmg

**Comment by TanteGerd** [comment]
The download works, and I can open the program, but when I transcribe a file there is no text coming out - what am I doing wrong?
